packages/block-library/src/social-link/index.php

```diff
@@ -21,7 +21,7 @@ function render_block_core_social_link( $attributes, $content, $block ) {
 
 	$service     = ( isset( $attributes['service'] ) ) ? $attributes['service'] : 'Icon';
 	$url         = ( isset( $attributes['url'] ) ) ? $attributes['url'] : false;
-	$label       = ( isset( $attributes['label'] ) ) ? $attributes['label'] : block_core_social_link_get_name( $service );
+	$label       = ( ! empty( $attributes['label'] ) ) ? $attributes['label'] : block_core_social_link_get_name( $service );
```

Comment:
Sorry! I had already created a [PR](https://github.com/WordPress/gutenberg/pull/60060) for this, with an extra check in edit.js. It's been merged, but I attributed you in the commit.

Reply:
@ramonjd yes I saw https://github.com/WordPress/gutenberg/pull/60060 this morning, thanks. 
It would be nice to better coordinate. Creating an issue before any non trivial PR, as required by the contributing guidelines of this project, would help avoid duplicated efforts. I don't see any issue associated to https://github.com/WordPress/gutenberg/pull/60060. Thanks.